resources/usa_garage/client.lua

```diff
@@ -86,6 +86,7 @@ locations = {
 	{x = -1916.8400878906, y = 2030.8366699219, z = 140.73731994629, noBlip = false}, -- Orange Farm
 	{x = 436.27728271484, y = 6529.9682617188, z = 27.877355575562, noBlip = false}, -- Orange Farm Paleto
 	{x = -181.43, y = -1289.32, z = 31.3, noBlip = true} -- Benny's
```

Comment:
```suggestion
	{x = -181.43, y = -1289.32, z = 31.3, noBlip = true}, -- Benny's
```

u guys really just love not testing ur changes and missing commas huh